clang/test/Driver/print-supported-extensions.c

```diff
@@ -4,14 +4,17 @@
 // RUN: %if aarch64-registered-target %{ %clang --target=aarch64-linux-gnu \
 // RUN:   --print-supported-extensions 2>&1 | FileCheck %s --check-prefix AARCH64 %}
 // AARCH64: All available -march extensions for AArch64
+// AARCH64:     Name                Description
 
 // RUN: %if riscv-registered-target %{ %clang --target=riscv64-linux-gnu \
 // RUN:   --print-supported-extensions 2>&1 | FileCheck %s --check-prefix RISCV %}
 // RISCV: All available -march extensions for RISC-V
+// RISCV:     Name                Version   Description
 
 // RUN: %if arm-registered-target %{ %clang --target=arm-linux-gnu \
 // RUN:   --print-supported-extensions 2>&1 | FileCheck %s --check-prefix ARM %}
 // ARM: All available -march extensions for ARM
+// ARM:     Name                Description
```

Comment:
Include the first line of the output as well, so we know it lines up (for all 3 architectures).

Don't use a `-NEXT:` check, just `ARM:` again. Since the extensions may get reordered later and we have no need to be strict.

Reply:
E.g.
```
ARM:     crc                 Enable support for CRC instructions
```
Then if we ever added `arc` it wouldn't matter if it came first, we're still checking the important things.

Reply:
This was covered in unit-test already.

Reply:
It is, and that's great, but, if someone decides to refactor this one day and suddenly they aren't using those functions anymore, we could miss it. It's unlikely but it's cheap to add a few lines here as well to cover that possibility.

Also, it's far easier for someone who is trying to understand the option to read this test file than it is to know that it calls 3 different methods in the background. So from a documentation point of view it also helps.

Reply:
I'll add the extra checks.